### ISSUE TITLE
Question regarding normalization

### ISSUE THREAD

**Issue by seokg** [dafei-qin/NFR_pytorch#6]
Hi, thanks for the great work.

I was going through the inference code and found "gradients_std.npy" is used in the dataset code.
What is "gradients_std.npy" for? Should these values be calculated for every 3D models that have different triangulation?

https://github.com/dafei-qin/NFR_pytorch/blob/e3553faa77f65240ec20167aec6e814473233890/datasets.py#L111-L129

**Comment by dafei-qin** [comment]
Thanks for your interest in our work! 

The normalization helps the neural network to fit those deformation gradients. These normalization factors are calculated by taking all training 3d models into account. At inference we assume the new 3d model has similar distributions to the training ones, so just use the same .npy from the training.

**Comment by seokg** [comment]
Thanks for the clarification.
